sphinx/conf.py

```diff
@@ -29,14 +30,14 @@
     "./",
     (
         "./examples.ipynb",
-        "../examples/00_toolkit_examples.ipynb",
-        "../examples/01a_qc_tz_unaware_data.ipynb",
-        "../examples/01b_qc_tz_aware_data.ipynb",
-        "../examples/02_plant_aep_analysis.ipynb",
-        "../examples/02b_augmented_plant_aep_analysis.ipynb",
-        "../examples/03_turbine_ideal_energy.ipynb",
-        "../examples/04_electrical_losses.ipynb",
-        "../examples/05_eya_gap_analysis.ipynb",
+        "../examples/00_intro_to_plant_data.ipynb",
+        # TODO: uncomment when the examples are updated and reincorporated
```

Comment:
I think the intro to the examples page should contain a brief explanation about what each example focuses on and the overall flow of the examples. I can work on this.

Reply:
I've started working towards this and added some placeholders that could be helpful for getting this going. Feel free to edit as desired/needed.

Reply:
Great, this is a good start!